appinventor/components/tests/com/google/appinventor/components/runtime/ClockTest.java

```diff
@@ -41,49 +41,49 @@ public void testGetMillis() throws Exception {
   }
 
   public void testAddYears() throws Exception {
-    assertEquals("Oct 11, 1943 9:30:00 AM", Clock.FormatDateTime(Clock.AddYears(tc, 2)));
+    assertEquals("Oct 11, 1943", Clock.FormatDate(Clock.AddYears(tc, 2), "MMM d, yyyy"));
   }
 
   public void testAddYears2() throws Exception {
-    assertEquals("Oct 11, 1931 9:30:00 AM", Clock.FormatDateTime(Clock.AddYears(tc, -10)));
+    assertEquals("Oct 11, 1931 9:30 AM", Clock.FormatDateTime(Clock.AddYears(tc, -10), "MM/dd/yyyy H:mm a"));
```

Comment:
This test (and others) are failing.   For example, this one gives 10/11/1931 rather than October 11, 1931.
And similar stuff for the other tests.